src/modules/ztsi/src/lib/Ztsi.cpp

```diff
@@ -197,10 +204,15 @@ int Ztsi::Get(const char* componentName, const char* objectName, MMI_JSON_STRING
                 document.SetInt(static_cast<int>(enabledState));
                 status = SerializeJsonObject(payload, payloadSizeBytes, maxPayloadSizeBytes, document);
             }
-            else if (0 == Ztsi::m_reportedServiceUrl.compare(objectName))
-            {
-                std::string serviceUrl = GetServiceUrl();
-                document.SetString(serviceUrl.c_str(), document.GetAllocator());
+            else if (0 == Ztsi::m_reportedMaxManualAttestationsPerDay.compare(objectName)){
```

Comment:
>)){ [](http://example.com/codeflow?start=88&length=3)

Please break and place { on a separate line. Here are elsewhere. The code for this project uses:

statement
{
  ...
}

rather than (not using this):

statement {
   ....
}